llvm/utils/UpdateTestChecks/common.py

```diff
@@ -151,6 +159,13 @@ def __call__(self, parser, namespace, values, option_string=None):
         metavar="REGEX",
         help="Exclude lines matching REGEX",
     )
+    filter_group.add_argument(
+        "--filter-all-after",
```

Comment:
The existing `--filter` is used to specify a filter to include lines matching it and `--filter-out` to exclude. Something like `--filter-out-after` may be more accurate?